tests/integration/transactions/test_did_delete.py

```diff
@@ -0,0 +1,31 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.it_utils import (
+    sign_and_reliable_submission_async,
+    test_async_and_sync,
+)
+from tests.integration.reusable_values import WALLET
+from xrpl.models.response import ResponseStatus
+from xrpl.models.transactions import DIDDelete, DIDSet
+
+_VALID_FIELD = "1234567890abcdefABCDEF"
+
+
+class TestDIDSet(IntegrationTestCase):
```

Comment:
This test name is confusing since this should be about DIDDelete
```suggestion
class TestDIDDelete(IntegrationTestCase):
```

Reply:
Oops, copy-paste typo